### PR TITLE
Add view to accept card token instead of full card details.

### PR DESCRIPTION
**Pull request by zemekeneng**
Hi @vikalpj and @akarambir,
Thanks for getting this repo updated for django 1.9 and python 3. This was very helpful!

I added a new url (w/ a view and a serializer) so that we can use stripe.js in the client, and avoid CC#'s ever hitting the server.

Any feedback is welcome.

Question: 
In the test suite there is https://github.com/Fueled/django-rest-framework-stripe/blob/master/payments/tests/test_views.py#L64. I don't see where this url comes from. Have you been able to run the test suite? Maybe it is something Mohammud was using or maybe it is an artifact from the original project.

### CODE REVIEW

**Comment by vikalpj** [comment]
Thanks @zemekeneng, for add the end point.

We haven't got much chance to dig into the test cases for now. The above code seems to be coming from https://github.com/Fueled/django-rest-framework-stripe/commit/0351cfe529adc9f4c59d21647067a925f1791073 and seems like it should be changed to `stripe-change-card`

File: payments/api/views.py [hunk 1]
```diff
@@ -120,6 +121,42 @@ def post(self, request, *args, **kwargs):
...
+        try:
+            serializer = self.serializer_class(data=request.data)
+
+            if serializer.is_valid():
+                validated_data = serializer.validated_data
+
+                customer = self.get_customer()
+
+                token = validated_data['token']
+                customer.update_card(token)
```
Comment (akarambir) [inline payments/api/views.py]: I think we should validate the token first as the underlying `customer.update_card` is not handling incoming error. Previously our own code was generating this token, so there was no need. But now, the token is coming from api. See here https://stripe.com/docs/api#retrieve_token

The below `try.. except stripe.CardError` also won't handle this. The error would be something like:

```python
InvalidRequestError: Request req_8tiPGLdRt: No such token: tok_invalid_test_token
```

**Review by karambir — CHANGES_REQUESTED** [review]
Just a minor fix needed. Thanks for PR :)

File: payments/api/views.py (hunk 1 above)
Comment (akarambir) [inline payments/api/views.py]: `StripeError` should handle this. Thanks

**Review by karambir — APPROVED** [review]
LGTM

**Pull request merged**